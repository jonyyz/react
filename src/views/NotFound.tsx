import * as React from "react";

import ErrorMessage from "../components/ErrorMessage";

const NotFound: React.FunctionComponent = () =>
  <ErrorMessage text="Oops!  Sorry, the page you requested was not found! (404)" />;

export default NotFound;
