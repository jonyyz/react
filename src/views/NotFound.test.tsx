import * as React from "react";
import { shallow } from "enzyme";

import NotFound from "./NotFound";

describe("NotFound", () => {
  test("should render", () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
