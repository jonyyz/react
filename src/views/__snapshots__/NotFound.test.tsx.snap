// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render 1`] = `
<Memo()
  text="Oops!  Sorry, the page you requested was not found! (404)"
/>
`;
